docs/testing/webdriverio/01-overview.md

```diff
@@ -0,0 +1,35 @@
+---
+title: WebdriverIO Overview
+sidebar_label: Overview
+---
+
+# Overview
+
+WebdriverIO is a progressive automation framework built to automate modern web and mobile applications. It simplifies the interaction with your app and provides a set of plugins that help you create a scalable, robust and stable test suite.
+
+Testing with WebdriverIO has the following advantages:
+
+- __Cross Browser Support__: WebdriverIO is designed to support all platforms, either on desktop or mobile. You can run tests on actual browser your users are using, including covering different versions of them.
+- __Real User Interaction__: Interaction with elements in WebdriverIO through the WebDriver protocol is much closer to native user-triggered interactions than what can be achieved with emulated DOM environments (such as JSDom or Stencil's own Mock-Doc).
+- __Web Platform Support__: Running tests in actual browsers allows you to tap into the latest Web Platform features for testing your components, often not available when using virtual DOM environments.
+
+## Set Up
+
+To get started with WebdriverIO, all you need to do is to run their project starter:
+
+```bash npm2yarn
+npm init wdio@latest .
+```
+
+This will initiate WebdriverIOs configuration wizard that walks you through the setup. Make sure you select the following options when walking through it:
+
+- __What type of testing would you like to do?__: select either `Component or Unit Testing - in the browser` if you are interested adding unit tests for your components or `E2E Testing - of Web or Mobile Applications` if you like to test your whole application (you can always add either of them later on)
```

Comment:
This section is a little tricky to read in one "block" of text on smaller screens:
<img width="813" alt="Screenshot 2024-02-08 at 8 56 51 AM" src="https://github.com/ionic-team/stencil-site/assets/1930213/22bf9bb9-0c5c-480b-9ce7-6d74b199be60">

What do you think about adding bullets for each option to break this up/introduce a little whitespace?
```suggestion
- __What type of testing would you like to do?__ Select either:
  - `Component or Unit Testing - in the browser` if you are interested adding unit tests for your components
  - `E2E Testing - of Web or Mobile Applications` if you like to test your whole application 
  
  You can always add either of them later on
```
Which produces something like:
<img width="496" alt="Screenshot 2024-02-08 at 9 01 04 AM" src="https://github.com/ionic-team/stencil-site/assets/1930213/4809db40-1cb4-430f-b8af-5c5d8f0194f9">